src/server/editionsPage.tsx

```diff
@@ -103,6 +105,7 @@ const buildHtml = (
         <body>
 			${body}
 			${renderToString(scripts)}
+			<script>${inlineJS}</script>
```

Comment:
Is it better to use `renderToString` here?  (I don't know the answer to that)

Reply:
Not sure either, will take a look and see the pros/cons

Reply:
This has been updated so inline js is an optional param for the Scripts component and therefore uses renderToString